docs/framework/migration-guide/versions-and-dependencies.md

```diff
@@ -29,6 +29,7 @@ Each version of the .NET Framework contains the common language runtime (CLR), t
   
 |.NET Framework version|CLR version|Features|Included in Visual Studio version|✓ Included in<br />+ Can be installed on<br />Windows|✓ Included in<br />+ Can be installed on<br />Windows Server|To determine installed .NET version|  
 |----------------------------|-----------------|--------------|---------------------------------------|----------------------------------------------------|-----------------------------------------------------------|-----------------------------------------| 
+|4.7.1|4|- Support for .NET Standard 2.0.<br/> - Support for configuration builders, allowing the creation of configuration files at runtime.<br/> - Runtime feature detection, which allows you to determine whether a predefined feature is supported in the runtime environment.<br/> - Serializable value tuples.<br/> - Improved performance for garbage collection.<br/> - Support for portable PDBs.<br/> - SHA-2 support for <xref:System.Messaging.Message.HashAlgorithm%2A?displayProperty=nameWithType>.<br/> - Support for ASP.NET cookie parsing with the <xref:System.Web.HttpCookie.TryParse%2A?displayProperty=nameWithType> method.<br/> - Support for SHA-2 hasing for ASP.NET forms authentication credentials.<br/> - Numerous accessibility improvements for app developers; see [What's new in accessibility in the .NET Framework](../whats-new/whats-new-in-accessibility.md).| | ✓  10 Fall Creators Update <br/> <br/> + 10 Creators Update <br/> + 10 Anniversary Update <br/> + 8.1 <br/> +7| + 2016 <br/> + 2012 R2 <br/> + 2012 <br/> + 2008 R2 SP1 |Use `Release` DWORD:<br/><br/> - 460798 (Windows 10 Creators Update) <br/> - 460805 (all other OS versions) <br/><br/> (see [instructions](../../../docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md)) 
```

Comment:
Will this be ready tomorrow?

Reply:
@vivmishra, no, it won't be. It should be ready at the end of the week. In the meantime, I'll link to Preeti's blog post.